packages/wrangler/src/d1/create.tsx

```diff
@@ -19,17 +20,23 @@ export function Options(yargs: CommonYargsArgv) {
 			type: "string",
 			demandOption: true,
 		})
+		.option("location", {
+			describe:
+				"A hint for the primary location of the new DB. Options:\nweur: Western Europe\neeur: Eastern Europe\napac: Asia Pacific\nwnam: Western North America\nenam: Eastern North America \n",
```

Comment:
These aren't open for discussion fwiw: https://developers.cloudflare.com/r2/buckets/data-location/#available-hints